README.md

```diff
@@ -76,12 +102,14 @@ options:
   -s S                  subpack config to use (eg: NO_WAVE)
 ```
 
+<br>
+
 ### Compile and build full shader pack
 ```
 ./build.sh pack
 ```
+
 The final mcpack will be inside `build/`.  
-**Command usage:**

```

Comment:
Why is this removed here, but not from the section above?

Reply:
mistake.